### ISSUE TITLE
Memory overloaded

### ISSUE THREAD

**Issue by Ged2323** [dgynfi/DYFStoreKit_Swift#2]
Hi,
When I go to use this for purchasing an auto-renewable product it seems to download a lot of data which tends to make my program crash.
Can you point me in the right direction to stop this.
Thanks in advance.
Regards Gerard

**Comment by itenfay** [comment]
Thank you for your feedback！

**Comment by Ged2323** [comment]
Do you know what could be causing this memory overload?

**Comment by itenfay** [comment]
@Ged2323 Hi, Gerard

`DYFStore` has been updated to version 2.0.2. Hosted content is default no longer supported.

This method is deprecated in SKPaymentQueue for iOS 16.
```
// Asynchronous.  Start the given downloads (SKDownload).
@available(iOS, introduced: 6.0, deprecated: 16.0, message: "Hosted content is no longer supported")
open func start(_ downloads: [SKDownload])
```
